src/driver/test/SecondaryAllowed.yml

```diff
@@ -0,0 +1,51 @@
+SchemaVersion: 2018-07-01
```

Comment:
I suspect you'll want some [QueryOptions](https://github.com/mongodb/genny/blob/master/src/driver/test/service-architecture/PrimaryRequiredReads.yml#L3)